docs/docs/external-volumes

```diff
@@ -0,0 +1,81 @@
+--------------------------------------------------------------
+title: Stateful Applications Using Persistent External Volumes
+--------------------------------------------------------------
+
+# Stateful Applications Using Persistent External Volumes
+
+<div class="alert alert-danger" role="alert">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> Adapted in Marathon Version 1.0 <br/>
+  The Persistent Storage functionality is considered beta, so use this feature at your own risk. We might add, change, or delete any functionality described in this document.
+</div>
+
+Marathon applications lose their state when they terminate and are relaunched. In some contexts, for instance, if your application uses MySQL, you’ll want your application to preserve its state. You can use an external storage service, such as Amazon's Elastic Block Store (EBS), to create a persistent volume that follows your application instance.
+
+Using an external storage service allows your apps to be more fault-tolerant. If a host fails, Marathon reschedules it on another host, along with its associated data, without user intervention.
+
+# Specifying an External Volume
+
+When you install DCOS, add the following to your `genconf.yml` file:
+
+- `rexray_config_method: file`
+
+- `rexray_config_filename: /path/to/rexray.yaml`
+
+<!-- [Learn more](URL). link to installer docs that mention this when they are live-->
+
+## Scaling your App
+
+Apps that use external volumes should only be scaled to a single instance.
+
+If you scale your app down to 0 instances, the volume is unattached from the agent where it was mounted, but it is not deleted. If you scale your app up again, the data that had been associated with it will still be available.
+
+## Create an Application with External Volumes
+
+You specify an external volume in the app definition of your Marathon app. [Learn more about Marathon application definitions](application-basics.html).
+
+```json
+{ 
+  "id": "hello",
+     "cmd": "while [ true ] ; do touch/var/lib/rexray/volumes/test12345/data/hello ; sleep 5 ; done",   
+     "mem": 32,   
+     "cpus": 0.1,   
+     "instances": 1,
+     "env": {
+          "DVDI_VOLUME_DRIVER": "rexray",
```

Comment:
for now this will work. however, specifying via envvar will be deprecated with a PR that I'm working on to enhance the persistent vol API in Marathon.